core/container.go

```diff
@@ -452,7 +457,7 @@ type ContainerExecOpts struct {
 }
 
 func (s *containerSchema) exec(ctx *router.Context, parent *Container, args containerExecArgs) (*Container, error) {
-	return parent.Exec(ctx, s.gw, s.platform, args.Args, args.Opts)
```

Comment:
I like that this went away, it felt a little odd that this was the only thing that needed it. (Turns out...)